Compile error: Cannot find module 'xterm-headless'

This happens after watch is run I think a couple of times:

![image](https://user-images.githubusercontent.com/2193314/130067630-c15659b7-74c7-43e3-a2e8-05cc9dfabec7.png)

Probably to do with us leaning on import `'xterm'` being the repo root but `'xterm-headless'` lives in `root/headless` and is generated.

I get this by doing the following:
```bash
git clone git@github.com:xtermjs/xterm.js
yarn
yarn build # successful
# <make some edit to src/browser/Terminal.ts>
yarn build # fails
```

Failure message:

```
stafford@DESKTOP-N6KOGQN:~/git/xterm2$ yarn build
yarn run v1.22.11
warning ../package.json: No license field
$ tsc -b ./tsconfig.all.json
out/headless/public/Terminal.d.ts:2:161 - error TS2307: Cannot find module 'xterm-headless' or its corresponding type declarations.

2 import { IBufferNamespace as IBufferNamespaceApi, IMarker, IModes, IParser, ITerminalAddon, ITerminalOptions, IUnicodeHandling, Terminal as ITerminalApi } from 'xterm-headless';

                                                            ~~~~~~~~~~~~~~~~

out/headless/public/Terminal.d.ts:2:161 - error TS2307: Cannot find module 'xterm-headless' or its corresponding type declarations.

2 import { IBufferNamespace as IBufferNamespaceApi, IMarker, IModes, IParser, ITerminalAddon, ITerminalOptions, IUnicodeHandling, Terminal as ITerminalApi } from 'xterm-headless';

                                                            ~~~~~~~~~~~~~~~~


Found 2 errors.

error Command failed with exit code 2.
```

--- a/addons/xterm-addon-serialize/benchmark/tsconfig.json
+++ b/addons/xterm-addon-serialize/benchmark/tsconfig.json
@@ -14,7 +14,7 @@
       "SerializeAddon": ["../src/SerializeAddon"]
     }
   },
-  "include": ["../**/*", "../../../typings/xterm.d.ts", "../../../out/**/*"],
+  "include": ["../**/*", "../../../typings/xterm.d.ts"],
   "exclude": ["../../../**/*test.ts", "../../**/*api.ts"],
   "references": [
     { "path": "../../../src/common" },

--- a/test/benchmark/tsconfig.json
+++ b/test/benchmark/tsconfig.json
@@ -21,8 +21,7 @@
   },
   "include": [
     "./**/*",
-    "../../typings/xterm.d.ts",
-    "../../out/**/*"
+    "../../typings/xterm.d.ts"
   ],
   "exclude": [
     "../../**/*test.ts"

